main.py

```diff
@@ -634,6 +634,46 @@ def run(self):
     return rid
 
 
+@app.post("/dds/att/")
+async def set_dds_attenuation(device: str, channel: int, value: float):
```

Comment:
Please annotate the return type. Also, note that it may return `None`. Or you could give for example -1 for the return value.

Reply:
As your suggestion, it looks better to return -1 instead of returing nothing.